src/GoogleCloudPubSubAdapter.php

```diff
@@ -185,8 +185,9 @@ public function setMaxMessages($maxMessages)
      *
      * @param string $channel
      * @param callable $handler
+     * @param bool $checkResponse 
      */
-    public function subscribe($channel, callable $handler)
+    public function subscribe($channel, callable $handler, $checkResponse = false)
```

Comment:
`$checkResponse` is a bad name, suggestions?

Reply:
As @matthewgoslett said we should rather try not to break the interface here..

Reply:
Done, however `checkResponse` is still a bad name, any ideas?

Reply:
Thanks for keeping me in check @matthewgoslett ❤️